script.js

```diff
@@ -0,0 +1,18 @@
+// HAMBURGER-MENU
+function mobileMenu() {
+  const window = document.querySelector('.hambu-wind');
+  window.classList.replace('not-active', 'active');
+}
+
+function closeMenu() {
+  const window = document.querySelector('.hambu-wind');
+  window.classList.replace('active', 'not-active');
+}
+
+document.getElementById('ham-link').addEventListener('click', mobileMenu);
+document.getElementById('close').addEventListener('click', closeMenu);
+const anchorElm = document.querySelectorAll('.pop-window');
+
+for (let i = 0; i < anchorElm.length; i += 1) {
+  anchorElm[i].addEventListener('click', closeMenu);
+}
```

Comment:
- [OPTIONAL] I noticed that you have exactly the same code in `DOM.js` file, so I think you don't need this file you can remove it. and in `about.html` file you can import/use  `DOM.js` instead 👍

> For Example
```
// About.html file
<script src="/DOM.js"> </script>
```
instead of 

```
// About.html file
<script src="/script.js"> </script>
```

> You don't have to write two js file 💡